src/data/reference-data/reference-aliases.ts

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash/fp';
+
+// A dictionary taking reference alias names to their backend unique name used in workflow inputs
+export const RefAliasToName = {
```

Comment:
A style convention: Names starting with upper case letters are usually reserved for classes / components. Suggest lower casing this name.

```suggestion
export const refAliasToName = {
```

Also, since other code tends to use the full "reference" vs abbreviating, I'd suggest doing the same here.

```suggestion
export const referenceAliasToName = {
```